src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

```diff
@@ -686,10 +686,11 @@ class Project: ADOSVTBase
     }
     
     hidden [ControlResult] CheckSecureFilesPermission([ControlResult] $controlResult) {
-        # getting the project ID
-        $projectId = ($this.ResourceContext.ResourceId -split "project/")[-1].Split('/')[0]
-        $url = "https://dev.azure.com/$($this.OrganizationContext.OrganizationName)/$($projectId)/_apis/distributedtask/securefiles?api-version=6.1-preview.1"
         try {
+            # getting the project ID
+            $controlResult.VerificationResult=[VerificationResult]::Failed;
+            $projectId = ($this.ResourceContext.ResourceId -split "project/")[-1].Split('/')[0]
+            $url = "https://dev.azure.com/$($this.OrganizationContext.OrganizationName)/$($projectId)/_apis/distributedtask/securefiles?api-version=6.1-preview.1"
             $response = [WebRequestHelper]::InvokeGetWebRequest($url);
```

Comment:
$response = **@(**[WebRequestHelper]::InvokeGetWebRequest($url)**)**;

If block can be updated to something like below:
if ( $response.Count -gt 0 -and [Helpers]::CheckMember($response [0],"id"))

Please try this